app/models/manageiq/providers/foreman/configuration_manager.rb

```diff
@@ -13,10 +13,16 @@ class ManageIQ::Providers::Foreman::ConfigurationManager < ManageIQ::Providers::
            :authentications,
            :connect,
            :endpoints,
+           :url,
+           :url=,
```

Comment:
We also need to delegate `endpoints=` and `authentications=` here, because otherwise editing them would not go through properly.

Reply:
:+1: done [here](https://github.com/ManageIQ/manageiq-providers-foreman/pull/62/files#diff-6d90565acdc2eee8687550e516a53b74R17) and [here](https://github.com/ManageIQ/manageiq-providers-foreman/pull/62/files#diff-6d90565acdc2eee8687550e516a53b74R14)